DTDL/siemens_industrial_edge_stream.json

```diff
@@ -0,0 +1,33 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:io:tributech:stream:siemens_industrial_edge;1",
+    "@type": "Interface",
+    "displayName": "Siemens Industrial Edge Stream",
+    "extends": "dtmi:io:tributech:stream:base;1",
+    "contents": [
+        {
+            "@type": "Property",
+            "name": "ConnectionName",
```

Comment:
The `ConnectionName` and  `CollectionName` will be the same for multiple streams since they are grouping elements on the Siemens side.
@simonpfeifhofer @mayrmax To we already have some other structure/element which we should use for such groupings? Or should we stick to the simple approach with out-of-the-box support by agent-companion in this scenario event if it means that the user has to enter same values for multiple streams?

Reply:
We could follow the approach of using relations. This would provide a benefit if we have multiple properties which are in use to define the group. It would be the case because we have `ConnectionName` and `CollectionName`. The tree-structure of the Agent-Companion or the Twin-Builder doesn't offer to option to configure it and the API would be necessary to configure it. Therefore I argue that it is fine to stick to the current solution and set `ConnectionName` and `CollectionName` to the same value for the entire group and add a Backlog-Item to optimize it in the future when the Agent-Companion and the Twin-Builder offer support for it. Makes sense?

Reply:
Yes I also see the mid-/long-term need to support grouping elements in the agent-companion UI.
Here it would be a multi-level hierarchy `Connection -> Collection -> DataPoints` for the grouping. The groups do not have any additional properties which are used at the moment on our end beside there name.

Since it doesn't exist yet I would stick to the simple approached as you agreed on.
@mbtributech Can you create the story mentioned by Simon?